spec/existing_project1_spec.rb

```diff
@@ -0,0 +1,112 @@
+require 'spec_helper'
+
+
+
+RSpec.describe "Create Existing Project with empty textboxes for details" do
+
+    let(:email) {"adneela@hotmail.com"}
+	let(:password) {"adnela"}
+	let(:project) {"Pro pro pro"}
+	let(:description) {"whhhy"}
+	let(:sponsor) {"neki@email.com"}
+	let(:responsibility) {"New respoo"}
+	let (:description) {"Descriptiooon"}
+	let (:ime) {"Adnela Cavcic"}
+	let(:login) { Login.new(@browser)}
+	let(:projekat) {Project.new(@browser)}
+
+
+
+
+	 
+    context "Login page" do
+	
+        it "Successful login" do
+	       login.login_user(email, password, ime)
+	       @browser.wait_until{@browser.url=="http://ah-test.abhapp.com/profile"}
+		   expect(@browser.text).to include(ime)
+			
+		end
+
+   end
+
+
+    context "Projects tab" do
+
+	   it "Switch to Projects Page" do
+
+          projekat.clickon_project 
+          @browser.wait_until{@browser.url == "http://ah-test.abhapp.com/projects"}
+		  expect(@browser.text).to include("All projects")
+	  
+	  end
+
+   end
+
+   context "Click on button Create Project button" do
+
+	   it "Page with two options to choose type of Project" do
+
+		  projekat.clickon_create_project
+		  @browser.wait_until{@browser.url == "http://ah-test.abhapp.com/projects/new"}
```

Comment:
Waiting for URL change is not a good practice. URL can be changed in a second, but content of the page can still be loading, and your test would fail. You should be waiting for particular elements to be present, rather than URL to change. Look up how to wait for element to be present in Watir.